src/components/home-block.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <ul class="row list-unstyled pt-4 text-center">
+    <li class="col" v-for="card in cards" :key="card.title">
```

Comment:
Actually we have no restriction that title has to be unique, so using it as key can be riscky